dev-tools/reconfigurator-cli/src/lib.rs

```diff
@@ -179,7 +183,26 @@ impl ReconfiguratorSim {
                     .add_omicron_zone_nic(zone.id, nic)
                     .context("adding omicron zone NIC")?;
             }
+
+            match &zone.zone_type {
```

Comment:
I suppose this is worth calling out - this means, while using the simulator, we "automatically move to the new nexus" when the top-level nexus_generation updates.

(The simulator exists independently of database state -- so basically, I'm making the choice that quiescing acts instantly from the simulator point-of-view. There shouldn't be anything in the blueprint to update anyway)